esp/mlrs-wireless-bridge-esp8266/mlrs-wireless-bridge-esp8266.ino

```diff
@@ -0,0 +1,287 @@
+//*******************************************************
+// mLRS Wireless Bridge for ESP8266
+// License: GPL v3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// Basic but effective & reliable transparent WiFi<->serial bridge
+//*******************************************************
+// 24. Aug. 2023
+//*********************************************************/
+// Adapted from mlrs-wireless-bridge for ESP32 by OlliW42
```

Comment:
pl drop the "by OlliW42". 
first, it can be missleading as it could be read as "adapted ... by..." and also the credential is already in teh other file.